graphs/kosaraju.cpp

```diff
@@ -1,19 +1,23 @@
 // Kosaraju - SCC O(V+E)
-
+// For undirected edges uncomment lines related to par, so after dfs it may know its route back through revdfs
```

Comment:
Acho mais direto só falar algo do tipo "For undirected graph, uncomment lines below", a pessoa vai entender o que fazer. Deixar ou não a explicação (so after...) aí é decisão sua msm, se achar que ajuda, pode deixar

Reply:
done